settings.gradle

```
@@ -0,0 +1,38 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * The settings file is used to specify which projects to include in your build.
+ *
+ * Detailed information about configuring a multi-project build in Gradle can be found
+ * in the user manual at https://docs.gradle.org/7.2/userguide/multi_project_builds.html
+ */
+
+plugins {
+    id "com.gradle.enterprise" version "3.2"
+}
+
+rootProject.name = 'module-ballerinai-observe'
+
+include ':checkstyle'
+include ':observe-native'
+include ':observe-ballerina'
+include ':integration-test-utils-native'
+include ':testobserve-ballerina'
+include ':integration-test-utils-tests'
+include ':observe-ballerina-integration-tests'
```

Shall we treat the modules as two sets "observe" modules set and "testobserve" modules set ? Then we can rename the modules and arrange them here accordingly.
```suggestion
include ':checkstyle'
include ':testobserve-native'
include ':testobserve-ballerina'
include ':testobserve-ballerina-tests'
include ':observe-native'
include ':observe-ballerina'
include ':observe-ballerina-integration-tests'
```
wdyt ?

---

Update as we discussed https://github.com/ballerina-platform/module-ballerinai-observe/pull/1/commits/4b2218280291e2dee542697481cd43e7f8469747